### ISSUE TITLE
Part2: Promise lingo and states vs fates

### ISSUE THREAD

**Issue by outbits** [fullstack-hy2020/fullstack-hy2020.github.io#2356]
Hello!

I noticed in part2 that the material could have been conflating __states__ and __fates__ of promises:
![image](https://user-images.githubusercontent.com/84378360/202441662-08c1844c-f72f-496a-a058-5e26fba23b60.png)

This could potentially trip people up down the line (It has been the case for me certainly, for some time, that's why I'm posting this), because rejected promises can also be unresolved, or resolved to a value or another promise. .

Maybe rewording things a bit, or redirecting to the [fates and states document in the original Promises proposal](https://github.com/domenic/promises-unwrapping/blob/master/docs/states-and-fates.md) for [more info](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise) is worthwhile?

**Comment by KartikAroraOfficial** [comment]
Hey, This would be updated and fixed with a PR, this seems like a valuable addition to make.

**Comment by vejol** [comment]
Thank you for this issue. I removed the mention about the term "resolved" from the material as it can cause confusion. Understanding the states of pending, fulfilled, and rejected is sufficient for the course purposes, and other terms are not crucial. I added a link to Mozilla's documentation, which describes promises and terms related to them in great detail. I consider this issue fixed now.
